### PR TITLE
Added nextflowVersion documentation

### PR DESCRIPTION
**Pull request by MaxUlysse**
- Add doc about manifest.nextflowVersion cf #752

### CODE REVIEW

File: docs/config.rst [hunk 1]
```diff
@@ -403,6 +403,20 @@ brackets. For example::
 To learn how to publish your pipeline on GitHub, BitBucket or GitLab code repositories read :ref:`sharing-page`
 documentation page.
 
+nextflowVersion
+^^^^^^^^^^^
+
```
Comment (pditommaso) [inline docs/config.rst]: Very good! RST format requires the `^^^` to have exactly the same length as the title.

Comment (MaxUlysse) [reply docs/config.rst]: ok, did not notice this error when I build the documentation on my computer.
I'll update it.
Thanks

Comment (pditommaso) [reply docs/config.rst]: Also, please use `Nextflow version` as title instead of the camel-case

Comment (pditommaso) [reply docs/config.rst]: oauch .. too late 😬

Comment (MaxUlysse) [reply docs/config.rst]: Np, I can make another commit if you want

Comment (pditommaso) [reply docs/config.rst]: let's do!

**Pull request merged**